internal/storage/metricstore/elasticsearch/testdata/test_call_rate.json

```diff
@@ -0,0 +1,109 @@
+{
```

Comment:
* `test_` prefix is redundant
* use more descriptive names, e.g. `query_***` or `output_***`